tests/integration/test_charm.py

```diff
@@ -112,7 +112,6 @@ async def test_monitor_user(ops_test: OpsTest) -> None:
     admin_mongod_cmd = await get_mongo_cmd(ops_test, unit.name)
     admin_mongod_cmd += f" {mongo_uri} --eval 'rs.conf()'"
     complete_command = f"ssh --container mongod {unit.name} {admin_mongod_cmd}"
-
```

Comment:
I think the spaces here make the code more readable, but that is just a preference, not a big deal

Reply:
I prefer to address that in a separate PR